src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[],
+  handleChangeCompleted: (id: number) => void,
+};
+
+export const TodosList: React.FC<Props> = ({
+  todos,
+  handleChangeCompleted,
+}) => {
+  return (
+    <>
+      {todos.map((todo) => (
+        <div
+          className={`todo ${todo.completed === true && ('completed')}`}
```

Comment:
Use logical operators for boolean values 
```suggestion
          className={`todo ${todo.completed && ('completed')}`}
```